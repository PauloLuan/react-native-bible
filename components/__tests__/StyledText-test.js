import { render } from '@testing-library/react-native'
import * as React from 'react'
import { MonoText } from '../StyledText'

describe(`Styled Text:`, () => {
  it(`renders correctly`, () => {
    const tree = render(<MonoText>Snapshot test!</MonoText>).toJSON()

    tree

    expect(tree).toMatchSnapshot()
  })
})
