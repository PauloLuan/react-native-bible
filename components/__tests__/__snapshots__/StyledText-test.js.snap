// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Text: renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000",
      },
      Array [
        undefined,
        Object {
          "fontFamily": "space-mono",
        },
      ],
    ]
  }
>
  Snapshot test!
</Text>
`;
